src/supporting-data/landing-page/LandingPage.tsx

```diff
@@ -0,0 +1,280 @@
+                }}
+              >
+                Taxonomy
+              </Link>
+            </h3>
+            <p>
+              Classification of the protein based on the organism of origin,
+              linking to databases like NCBI Taxonomy.
+            </p>
+          </div>
+          {/* Cross-referenced Databases */}
+          <div className="uniprot-grid-cell--small-span-6 uniprot-grid-cell--medium-span-4">
+            <h3>
+              <Link
+                to={{
+                  pathname: LocationToPath[Location.DatabaseResults],
+                  search: 'query=*',
+                }}
+              >
+                Cross-referenced Databases
+              </Link>
+            </h3>
+            <p>
+              Links to databases like PDB (Protein Data Bank), Gene Ontology
+              (GO), Ensembl, and RefSeq.
+            </p>
+          </div>
+          {/* Human Diseases */}
+          <div className="uniprot-grid-cell--small-span-6 uniprot-grid-cell--medium-span-4">
+            <h3>
+              <Link
+                to={{
+                  pathname: LocationToPath[Location.DiseasesResults],
+                  search: 'query=*',
+                }}
+              >
+                Human Diseases
+              </Link>
+            </h3>
+            <p>
+              Provides insights into the relationship between proteins and human
+              health.
+            </p>
+            <p>
+              Includes disease annotations, pathogenesis information, variants
+              linked to diseases, and links to drug databases.
+            </p>
+          </div>
+          {/* Automatic Annotations */}
+          <div className="uniprot-grid-cell--small-span-12 uniprot-grid-cell--medium-span-6">
+            <h2>Automatic Annotations</h2>
+            <p>
+              The Automatic Annotations section in UniProt provides functional
+              and structural insights for protein sequences and supplements
+              manually curated data using two key systems: UniRule and ARBA.
+            </p>
+            <p>
+              <Link to={LocationToPath[Location.UniRuleResults]}>UniRule</Link>:
+              A manually curated rule-based system that delivers high-confidence
+              annotations for well-characterized protein families based on
+              experimental evidence and domain information.
+            </p>
+            <p>
+              <Link to={LocationToPath[Location.ARBAResults]}>ARBA</Link>: An
+              automated system that uses computationally generated rules to
+              annotate a broader range of protein sequences, including those
+              with less-established functional data.
+            </p>
+          </div>
+          {/* Data integration form */}
+          <div className="uniprot-grid-cell--small-span-12 uniprot-grid-cell--medium-span-6">
+            <h2>Expand Your Reach: Connect Your Database with UniProt</h2>
```

Comment:
Is it possible to push the "Expand Your Reach: Connect Your Database with UniProt" below the automatic annotations to distinguish it from the rest of the namespaces? At the same time, the form seems related to Cross-referenced Databases so is there a way we could move these two things closer in the page or somehow indicate a relationship?